components/brave_referrals/browser/BUILD.gn

```diff
@@ -18,6 +16,8 @@ source_set("browser") {
       "//brave/common",
       "//brave/components/brave_referrals/common",
       "//brave/components/brave_stats/browser",
+      "//brave/components/private_channel",
```

Comment:
all usages of private_channel need to be behind a buildflag

Reply:
- [x] add buildflag